app/src/main/java/science/apolline/utils/DataExport.kt

```diff
@@ -67,6 +67,7 @@ object DataExport : AnkoLogger {
                 val fw = FileWriter(createFileName("json"))
                 val gson = GsonBuilder().setPrettyPrinting().create()
                 val jsonFile = gson.toJson(dataList)
+                print(jsonFile.toString())
```

Comment:
Is this instruction really necessary or a past debugging instruction you introduced?

Reply:
It is not necessary, it is indeed a debugging instrcution that I forgot to take away

Reply:
Please remove it and deliver a code as clean as possible

Reply:
ok thanks